ci/kokoro/release/publish.cfg

```diff
@@ -0,0 +1,8 @@
+build_file: "google-cloud-cpp/.kokoro/release/publish.sh"
```

Comment:
Should this now be `google-cloud-cpp/ci/kokoro/release/publish.sh`?

Reply:
Yes.  Fixed.